ingestion/tests/integration/postgres/test_postgres.py

```diff
@@ -204,3 +208,96 @@ def test_profiler(run_profiler_workflow):
 
 def test_lineage(ingest_lineage):
     pass
+
+
+def run_usage_workflow(db_service, metadata):
+    workflow_config = {
+        "source": {
+            "type": "postgres-usage",
+            "serviceName": db_service.fullyQualifiedName.__root__,
+            "serviceConnection": db_service.connection.dict(),
+            "sourceConfig": {
+                "config": {"type": DatabaseUsageConfigType.DatabaseUsage.value}
+            },
+        },
+        "processor": {"type": "query-parser", "config": {}},
+        "stage": {
+            "type": "table-usage",
+            "config": {
+                "filename": "/tmp/postgres_usage",
+            },
+        },
+        "bulkSink": {
+            "type": "metadata-usage",
+            "config": {
+                "filename": "/tmp/postgres_usage",
+            },
+        },
+        "sink": {"type": "metadata-rest", "config": {}},
+        "workflowConfig": {
+            "loggerLevel": "DEBUG",
+            "openMetadataServerConfig": metadata.config.dict(),
+        },
+    }
+    workflow = UsageWorkflow.create(workflow_config)
+    search_cache.clear()
+    workflow.execute()
+    workflow.raise_from_status()
+
+
+@pytest.mark.xfail(
+    reason="'metadata.ingestion.lineage.sql_lineage.search_cache' gets corrupted with invalid data."
+    " See issue https://github.com/open-metadata/OpenMetadata/issues/16408"
```

Comment:
ah got it, we just want this to fail for now and fix it later

Reply:
yes. this way we can deliver a test before delivering the fix.